src/consts.js

```diff
@@ -4,16 +4,16 @@ export const RECONNECT_SECONDS = 15
 
 export const APP_VERSION = process.env.VUE_APP_UI_VERSION
 
-export const DNA_VERSION = '0_1_0_alpha3'
-export const DNA_UUID = 'develop'
+export const DNA_VERSION = process.env.VUE_APP_DNA_VERSION
+export const DNA_UID = process.env.VUE_APP_DNA_UID
 
 export const INSTALLED_APP_ID = process.env.VUE_APP_INSTALLED_APP_ID
   ? process.env.VUE_APP_INSTALLED_APP_ID
   : process.env.VUE_APP_WEB_CLIENT_PORT === '8888' // for development/testing: dev agent 1 is served at port 8888, and dev agent 2 at port 9999
     ? 'elemental-chat-1'
     : process.env.VUE_APP_WEB_CLIENT_PORT === '9999'
       ? 'elemental-chat-2'
-      : `elemental-chat:${DNA_VERSION}${DNA_UUID ? ':' + DNA_UUID : ''}` // default to elemental-chat:<dna version number>:<uuid> (appId format for holo self-hosted)
+      : `elemental-chat:${DNA_VERSION}${DNA_UID ? ':' + DNA_UID : ''}` // default to elemental-chat:<dna version number>:<uuid> (appId format for holo self-hosted)
```

Comment:
There's two colons now
```suggestion
      : `elemental-chat:${DNA_VERSION}${DNA_UID ? '::' + DNA_UID : ''}` // default to elemental-chat:<dna version number>:<uuid> (appId format for holo self-hosted)
```